locales/en.default.schema.json

```diff
@@ -1344,6 +1344,18 @@
           },
           "label": "Content layout"
         },
+        "animations": {
+          "content": "Animations"
+        },
+        "image_behavior": {
+          "options__1": {
+            "label": "None"
+          },
+          "options__2": {
+            "label": "Ambient movement"
+          },
+          "label": "Image behavior"
+        },
```

Comment:
As noted in https://github.com/Shopify/dawn/pull/2383#discussion_r1134129110, it might make sense to have these strings be global since they're re-used across a few Sections.